controllers/apps/components/internal/component_base_stateful.go

```diff
@@ -419,6 +423,25 @@ func (c *StatefulComponentBase) updatePVCSize(reqCtx intctrlutil.RequestCtx, cli
 	return nil
 }
 
+// statusUpdateTemplateRenderedConfig transform the action of rendered config by config-template from noop to update in cluster Status reconciliation
+func (c *StatefulComponentBase) statusUpdateTemplateRenderedConfig() error {
```

Comment:
@free6om this package has missing UT setup, it's bad that more codes are adding to this package, plz do resolve tech. debt ASAP.